src/cmd/version/version.go

```diff
@@ -0,0 +1,84 @@
+package version
+
+import (
+	"context"
+	cloudclient "github.com/otterize/otterize-cli/src/pkg/cloudclient/restapi"
+	"github.com/otterize/otterize-cli/src/pkg/config"
+	"github.com/otterize/otterize-cli/src/pkg/utils/prints"
+	"github.com/spf13/cobra"
+)
+
+var Version string
+var Commit string
+
+var Cmd = &cobra.Command{
+	Use:          "version",
+	Aliases:      []string{"api-version"},
+	Short:        "Get the Otterize CLI and API versions",
```

Comment:
All "API" should be "Cloud API" to clarify it's Otterize Cloud's API, not e.g. the network mapper's API.